packages/api/src/routers/api/alerts.ts

```diff
@@ -71,6 +81,55 @@ const validateGroupBy = async (req, res, next) => {
 };
 
 // Routes
+router.get('/', async (req, res, next) => {
+  try {
+    const teamId = req.user?.team;
+    if (teamId == null) {
+      return res.sendStatus(403);
+    }
+    const alerts = await Alert.find({ team: teamId });
+    const alertsWithHistory = await Promise.all(
+      alerts.map(async alert => ({
+        ...alert,
+        history: await getHistory(alert._id.toString(), teamId.toString()),
```

Comment:
nit: This shouldn't matter too much now, but perf could be improved with an `$in` query instead right?

Reply:
Yes I'm just writing completely brainless queries until I get the format and data I need, then I will do a pass (or ask someone with better Mongo knowledge 🤔 )